kostyor/resources/upgrades.py

```diff
@@ -0,0 +1,65 @@
+from flask import request
+from flask.ext.restful import Resource, fields, marshal_with, abort
+
+from kostyor.db import api as db_api
+
+
+_PUBLIC_ATTRIBUTES = {
+    'id': fields.String,
+    'cluster_id': fields.String,
+    'from_version': fields.String,
+    'to_version': fields.String,
+    'status': fields.String,
+    'upgrade_start_time': fields.DateTime('iso8601'),
+    'upgrade_end_time': fields.DateTime('iso8601'),
+}
+
+
+class Upgrades(Resource):
+
+    @marshal_with(_PUBLIC_ATTRIBUTES)
+    def get(self, cluster_id):
+        return db_api.get_upgrades(cluster_id)
+
+    @marshal_with(_PUBLIC_ATTRIBUTES)
+    def post(self, cluster_id):
+        payload = request.get_json()
+        upgrade = db_api.create_cluster_upgrade(
+            cluster_id,
+            payload['to_version']
+        )
```

Comment:
@jvarlamova Ha! That's a good catch. Thank you. :) I forgot to backport that part of the code.